app/menu.js

```diff
@@ -146,6 +146,11 @@ const template = [{
   label: 'Help',
   role: 'help',
   submenu: [{
+    label: 'About Orion',
```

Comment:
I guess macOS Users are expecting the About to be under the "Orion" Menu (as it was).

Can we add About Orion here just for Win+Gnu and under the Mac-only menu for Mac?